database/migrations/20230613204013-initial-schema.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(t => {
+      return Promise.all([
+        queryInterface.createTable('User', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          username: Sequelize.DataTypes.STRING,
+          email: Sequelize.DataTypes.STRING,
+          password: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Team', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          owner_user_id: {
+            type: Sequelize.DataTypes.BIGINT,
+          },
+          name: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Player', {
+          id: {
+            type: Sequelize.DataTypes.BIGINT,
+            autoIncrement: true,
+            primaryKey: true,
+          },
+          name: Sequelize.DataTypes.STRING,
+        }, { transaction: t }),
+        queryInterface.createTable('Year', {
```

Comment:
We generally don't retain data from year to year.  I guess that could be useful but I think it would be fine to exclude Year from the database.

Reply:
I'm currently using it for multiyear deals